subprojects/xcfa/xcfa-cli/src/main/java/hu/bme/mit/theta/xcfa/cli/XcfaCli.java

```diff
@@ -204,6 +206,12 @@ public class XcfaCli {
 	@Parameter(names = "--algorithm", description = "Algorithm to use when solving multithreaded programs")
 	XcfaConfigBuilder.Algorithm algorithm = XcfaConfigBuilder.Algorithm.SINGLETHREAD;
 
+	@Parameter(names = "--por", description = "Enable/disable partial order reduction for interleaving based multithreaded programs")
```

Comment:
I think this should be a new algorithm (see --algorithm) instead

Reply:
Done, see [XcfaConfigBuilder.Algorithm.INTERLEAVINGS_POR](https://github.com/csanadtelbisz/theta/blob/8a2a3305c0a88c95a622330b43e8acd4e26f5062/subprojects/xcfa/xcfa-analysis/src/main/java/hu/bme/mit/theta/xcfa/analysis/common/XcfaConfigBuilder.java#L164). ComplexPortfolio has also been updated.
POR now works with the ```--algorithm INTERLEAVINGS_POR``` (even when complex portfolio is used).